app/assets/stylesheets/articles.scss

```diff
@@ -1844,3 +1809,61 @@
   opacity: 0.7;
   display: none;
 }
+
+// Left Side bar navigation links
+.sidebar-nav-element {
```

Comment:
Pulled this out because it was deeply nested and could not be used in Storybook without creating a tonne of dummy elements to mimic the nesting.